sys/sysmon/ThreadStatus.cpp

```diff
@@ -12,6 +12,8 @@ namespace sysmon {
 
 ThreadStatus::ThreadStatus() : MemoryStatus() {
     pid_ = 0;
+    loops_ = 0;
+    overshots_ = 0;
 }
```

Comment:
Note that you could just write in the header:
```cpp
ThreadStatus::ThreadStatus() : MemoryStatus(), pid_(0), loops_(0), overshots_(0) {};
```

Reply:
done